packages/components/src/components/expansion-panel/array-renderer.jsx

```diff
@@ -0,0 +1,53 @@
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import React from 'react';
+import Icon from '../icon';
+
+const ArrayRenderer = (array, open_ids, setOpenIds) => {
```

Comment:
This is a component and should be used as such, i.e. `<ArrayRenderer your_props="go_here" />` vs. `ArrayRenderer()`

Reply:
True, I forgot to change it before. Fixed.